tests/correctness/numeric-types/fixed-point/binary-operators.expect

```diff
@@ -0,0 +1,26 @@
+{
+  "lhs": [
+    "0.125"
+  ],
+  "rhs": [
+    "4.656612873077392578125E-10"
```

Comment:
Dumb thing but what if you make this number be `4.999999...E-10` so that in the add case it's obvious that this  number is appended after the LHS value? It took me a minute to parse the `E-10`. Either way, your call.